src/cocaine-app/__init__.py

```diff
@@ -214,47 +212,48 @@ def init_minions():
     return m
 
 
-def init_planner(job_processor, niu, namespaces_settings, move_planner, external_storage_converting_planner, lrc_reserve_group_planner):
+def init_planner(job_processor, niu, namespaces_settings):
```

Comment:
Done in https://github.com/yandex/mastermind/pull/417/commits/879ce74554565fdfc93ed5cf8a914b829944d6d8